src/main/java/io/confluent/castle/cloud/DockerCloud.java

```diff
@@ -99,6 +99,15 @@ public String startup(CastleCluster cluster, CastleNode node,
             run.add(String.format("%s:%s", cluster.conf().kafkaPath(),
                 ActionPaths.KAFKA_SRC));
         }
+        if (!cluster.conf().schemaRegistryPath().isEmpty()) {
+            if (!Files.isDirectory(Paths.get(cluster.conf().schemaRegistryPath()))) {
```

Comment:
Can we just call `validateSchemaRegistryPath` here?

Reply:
Ack